toolchain/check/member_access.cpp

```diff
@@ -356,9 +397,10 @@ static auto LookupMemberNameInScope(Context& context, SemIR::LocId loc_id,
   return member_id;
 }
 
-// Performs the instance binding step in member access. If the found member is a
-// field, forms a class member access. If the found member is an instance
-// method, forms a bound method. Otherwise, the member is returned unchanged.
+// Performs the instance binding step in member access. If the found member
```

Comment:
Something funny seems to have happened here: the comments from here on in this file have all been reflowed to fit in about 76 columns. (I wonder if there was an unclosed open brace or two, and the whole file got reformatted like that, and the formatting tool is then not able to undo it because it only reflows overly-long comment lines?)

Reply:
Oops, yeah surely I did that, thanks for pointing it out.